space2stats_api/tests/test_api.py

```diff
@@ -0,0 +1,77 @@
+from fastapi.testclient import TestClient
```

Comment:
should we run the tests in the CI?

Reply:
yeah, you written nice tests they deserve to be ran in CI 😄

Reply:
First version implemented: https://github.com/worldbank/DECAT_Space2Stats/actions/runs/9999419959!